client/apps/edit/components/edit_container.jsx

```diff
@@ -48,7 +48,7 @@ export class EditContainer extends Component {
       lastUpdated: null,
       isOtherUserInSession: !!props.currentSession && !isCurrentUserEditing,
       inactivityPeriodEntered: false,
-      shouldShowModal: true,
+      shouldShowModal: !props.article.isNew(),
       sentStopEditingEvent: false
```

Comment:
FYI we will be using a plain object in the redux store v soon (after all the redux changes are merged), so it is probably better not to use `article.isNew` and instead check if the id exists

Reply:
Yeah I was thinking about that as well, let's switch it when we replace the backbone model for a plain object